app/models/podcast_appearance.rb

```diff
@@ -0,0 +1,11 @@
+class PodcastAppearance < ApplicationRecord
+  belongs_to :user
+  belongs_to :podcast_episode
+  validates :podcast_episode_id, uniqueness: { scope: :user_id }
+  validates :podcast_episode_id, :user_id, :role, presence: true
+  validates :role, inclusion: { in: %w[owner host guest], message: "%<value> is not a valid role" }
+
+  def podcast_episode_creator_id
+    podcast_episode.podcast.creator_id
+  end
```

Comment:
as this seems to be only necessary in the policy, I wouldn't add it as a method here

Reply:
as mentioned, this should be removed as it was only used for the policy